### PR TITLE
Add a setting to configure the audible bell

### LINKED ISSUE #2360: Feature Request: disable bell
# Description of the new feature/enhancement

Make a setting to disable the bell. This could be a per profile setting.

Maybe other "bell" settings can be desirable like:

```
"bell": "enable"  (default)
"bell": "disable" (this feature request)
"bell": "visual"
```

So with that in mind don't let the setting be a boolean.

Probably related to #72

### PR DESCRIPTION
**Pull request by zadjii-msft**
## Summary of the Pull Request

Adds a new setting, `bellStyle`, to be able to disable the audible bell added in #7679. Currently, this setting accepts two values:
* `audible`: play a noise on a bell
* `none`: Don't play a noise.

In the future, we can add a `"bellStyle": "visible"` for flashing the Terminal instead of making a noise on bell.

## References
* almost certainly going to explode on merge with #7667 

## PR Checklist
* [x] Closes #2360
* [x] I work here
* [ ] Tests added/passed
* [x] Requires documentation to be updated - https://github.com/MicrosoftDocs/terminal/pull/152

## Detailed Description of the Pull Request / Additional comments

**FOR DISCUSSION**: Should `bellStyle` just be `bell`?

## Validation Steps Performed
Pressing <kbd>Ctrl+G</kbd> in cmd, and hitting enter is an easy way of triggering a bell. I set the setting to `none`, and presto, the bell stopped.

### CODE REVIEW

File: src/cascadia/TerminalApp/GlobalAppSettings.h [hunk 1]
```diff
@@ -76,6 +76,7 @@ namespace winrt::TerminalApp::implementation
         GETSET_PROPERTY(bool, StartOnUserLogin, false);
         GETSET_PROPERTY(bool, AlwaysOnTop, false);
         GETSET_PROPERTY(bool, UseTabSwitcher, true);
+        GETSET_PROPERTY(winrt::TerminalApp::BellStyle, BellStyle, winrt::TerminalApp::BellStyle::Audible);
```
Comment (DHowett) [inline src/cascadia/TerminalApp/GlobalAppSettings.h:79]: 🚨 Carlos, this will be a conflict for you

**Review by DHowett — APPROVED** [review]
Perfect. Do we want it to be per-profile?

This will have an interesting effect on some applications, and users may be confused by it. There's some applications that _emit their own bells_ using the same API, and people may be upset that we can't suppress them.

Not a complaint, just something to be aware of!

**Comment by DHowett** [comment]
This could eventually be a _flag mapper_ (`bellStyle: [audible, visible]`)

**Review by carlos-zamora — CHANGES_REQUESTED** [review]
Update the docs and schema!

Also, please make my life easier and wait until after #7667 merges.

**Review by miniksa — APPROVED** [review]

File: doc/cascadia/profiles.schema.json [hunk 2]
```diff
@@ -717,6 +724,11 @@
           ],
           "type": "string"
         },
+        "bellStyle": {
+          "default": "audible",
+          "description": "Controls what happens when the application emits a BEL character. When set to \"audible\", the Terminal will play a sound. When set to \"none\", nothing will happen.",
+          "$ref": "#/definitions/BellStyle"
```
Comment (DHowett) [inline doc/cascadia/profiles.schema.json:730]: i'm glad it caught this b/c it wouldn't have been a valid schema otherwise!@ :wink:

**Review by DHowett — APPROVED** [review]

**Pull request merged**